src/docfx/restore/lock/DependencyLock.cs

```diff
@@ -3,16 +3,36 @@
 
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace Microsoft.Docs.Build
 {
     internal class DependencyLock : DependencyVersion
```

Comment:
can we have a pure data model that represents the content of .lock file, just like how we model other data files like xrefmap, publish manifest, git commits time etc.

Reply:
i splitted the model, will clean the constructor in another PR